manager/install.sh

```diff
@@ -238,6 +238,7 @@ function main() {
   echo "✓"
 
   echo -n "￮ configuring logging "
+  export CORTEX_TAGS_JSON=$(echo $CORTEX_TAGS  | jq -sR '[split(",")[] | rtrimstr("\n") | split("=") | {(.[0]): .[1]} ] | add' | tr -d '\n' )
```

Comment:
Nice job with the `jq` query! I have always struggled to write those :)

There is a simpler place we can do this, which is in `cluster_config_env.py` (that is the file which exports these env vars. Right below `print(f"export CORTEX_TAGS={inlined_tags}")`, we should be able to add a line like `print(f"export CORTEX_TAGS_JSON='{json.dumps(value)}'")`, what do you think about this approach?

Reply:
everything is better than jq lol